src/vorta/views/repo_add_dialog.py

```diff
@@ -208,6 +211,24 @@ def password_listener(self):
             self.passwordLabel.setText(translate('utils', msg))
             return not bool(msg)
 
+    def cancel_job(self):
+        self.close()
+        try:
+            if self.job:
+                self.job.cancel()
```

Comment:
There is no `self.job` currently.

Reply:
Hm! I was wondering how it is still canceling the job 🤔

Reply:
> There is no `self.job` currently.

Hm! the close() method is working!, but I also noticed that you have to cancel the job within a specified time gap it might vary from machine to machine i guess, but after that time gap it adds the repo anyways

Reply:
> You are n't setting a meaningful status message yet which was requested in #1306.

![Screenshot from 2023-04-21 23-10-59](https://user-images.githubusercontent.com/34976093/233701075-019fac62-ef0a-4b2b-aa15-b075c19cfda9.png)

Done 👍🏼